tests/system/action/assignment/test_create.py

```diff
@@ -131,20 +139,61 @@ def test_create_full_fields(self) -> None:
                 "phase": "search",
                 "default_poll_description": "text_test2",
                 "number_poll_candidates": True,
+                "attachment_ids": [1],
             },
         )
         self.assert_status_code(response, 200)
-        model = self.get_model("assignment/1")
-        assert model.get("title") == "test_Xcdfgee"
-        assert model.get("meeting_id") == 110
-        assert model.get("description") == "text_test1"
-        assert model.get("open_posts") == 12
-        assert model.get("phase") == "search"
-        assert model.get("default_poll_description") == "text_test2"
-        assert model.get("number_poll_candidates") is True
-        agenda_item = self.get_model("agenda_item/1")
-        self.assertEqual(agenda_item.get("meeting_id"), 110)
-        self.assertEqual(agenda_item.get("content_object_id"), "assignment/1")
+        self.assert_model_exists(
+            "assignment/1",
+            {
+                "title": "test_Xcdfgee",
+                "meeting_id": 110,
+                "description": "text_test1",
+                "open_posts": 12,
+                "phase": "search",
+                "default_poll_description": "text_test2",
+                "number_poll_candidates": True,
+                "attachment_ids": [11],
```

Comment:
Please rename this `attachment_ids` to `meeting_mediafile_ids` to reflect also the collection name

Reply:
Done something like that